src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthCountEmptySmallSize.java

```diff
@@ -10,11 +10,11 @@
 
 public class InputMethodLengthCountEmptySmallSize {}
 class AA {
-    AA() { // ok, 2 lines
+    AA() {
```

Comment:
We are only requesting to remove "// ok" and not "// ok, _ _ _"

Reply:
Thanks for pointing that out, I've made the appropriate changes